htdocs/install/mysql/tables/llx_accounting_category_link.key.sql

```diff
@@ -0,0 +1,26 @@
+-- ===================================================================
```

Comment:
For table that are link tables, we used to use the name of both joined tables into the name of table link.
So can you name it 
llx_accounting_category_account-accounting.sql
so
llx_modulename_table1_table2-modulename.sql

Reply:
Done. Thanks @eldy